File: C LANGUAGE HARRY/CHAPTER 3 PRACTICE SET/problem_2.c
/*Write a program to determine whether a student has passed or failed. To pass, a
student requires a total of 40% and at least 33% in each subject. Assume there
are three subjects and take the marks as input from the user.*/

#include <stdio.h>
int main(){

    int marks1 , marks2, marks3;
    printf("ENTER MARKS1\n");
    scanf("%d", &marks1);

    printf("ENTER MARKS2\n");
    scanf("%d", &marks2);

    printf("ENTER MARKS3\n");
    scanf("%d", &marks3);
 
 printf("THE MARKS ARE : %d %d and %d\n", marks1,marks2,marks3);

 if (marks1<33||marks2<33||marks3<33){
    printf("YOU ARE FAIL IN INDIVIUAL SUBJECT(S)");
 }

 else if ((marks1+marks2+marks3)/3 <40){
    printf("you are fail ");
 }
else {
    printf("BALLE BALLE");
}




    return 0;
}

File: PYTHON/CHAPTER 6/problem 2.py
# Write a program to find out whether a student has passed or failed if it requires a
# total of 40% and at  least 33% in each subject to pass. Assume 3 subjects and
# take marks as an input from the user.

science = int(input("enter the marks "))
english = int(input("enter the marks "))
maths = int(input("enter the marks "))

percentage = (science + maths + english)/3.0

if (percentage > 40 and science > 33 and maths > 33 and english > 33) :
    print("you passed")

else :
    print("u failed")
